src/Microsoft.ML.Dnn/ImageClassificationTransform.cs

```diff
@@ -843,6 +839,7 @@ public void UpdateCacheIfNeeded()
             protected override Delegate MakeGetter(DataViewRow input, int iinfo, Func<int, bool> activeOutput, out Action disposer)
             {
                 disposer = null;
+                _parent._session.graph.as_default();
```

Comment:
>_parent._session.graph.as_default(); [](start = 16, length = 36)

not needed.

Reply:
Required for the Microsoft.ML.Scenarios.TensorFlowScenariosTests.TensorFlowImageClassification

---
In reply to: [324890773](https://github.com/dotnet/machinelearning/pull/4205#discussion_r324890773) [](ancestors = 324890773)